### PR TITLE
Wm/pg blog create

### PR DESCRIPTION
**Pull request by wmosienski**

### CODE REVIEW

File: database/procedures/author/create.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    country_id INT,
+    firstname CHAR(50),
+    lastname CHAR(50)
+)
+LANGUAGE plpgsql    
+AS $$
+BEGIN
+    INSERT INTO AUTHOR(COUNTRY_ID, FIRSTNAME, LASTNAME)
+    VALUES(country_id, firstname, lastname);
+    COMMIT;
```
Comment (Mikalai-Kastsevich-ITechArt) [inline database/procedures/author/create.sql]: Why do you use COMMIT here?

Comment (Mikalai-Kastsevich-ITechArt) [reply database/procedures/author/create.sql]: Usualy functions that create new entities shoud return added entity id

File: database/tables/author.table.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS AUTHOR CASCADE;
+
+CREATE TABLE AUTHOR(
+   ID         SERIAL PRIMARY KEY      NOT NULL,
```
Comment (Mikalai-Kastsevich-ITechArt) [inline database/tables/author.table.sql]: You can check but I think there is no need to use NOT NULL for PRIMARY KEY, cause it's cannot be nullable

File: database/functions/author/create.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    firstname CHAR(50),
+    lastname CHAR(50)
+) RETURNS INT
+LANGUAGE plpgsql    
+AS $$
+DECLARE res INT;
+BEGIN
+    INSERT INTO AUTHOR(COUNTRY_ID, FIRSTNAME, LASTNAME)
+    VALUES(country_id, firstname, lastname)
+    RETURNING ID INTO res;
```
Comment (Mikalai-Kastsevich-ITechArt) [inline database/functions/author/create.sql:12]: I dont's thin that use variable of res is needed here, you can just use RETURNING ID

File: database/procedures/author/create.sql (hunk 1 above)
Comment (wmosienski) [reply database/procedures/author/create.sql]: addressed

File: database/functions/author/create.sql (hunk 3 above)
Comment (wmosienski) [reply database/functions/author/create.sql:12]: can't come up with the working solution without a variable :(

**Pull request merged**